inventory/routes/concepts/use-routes.rst

```diff
@@ -0,0 +1,388 @@
+
+Rules
+~~~~~
+
+Logically, the rules are to be defined on the routes. To do so, go to
+:menuselection:`Inventory --> Configuration --> Routes`. There, you
+can find a *Rules* section featuring a *Add a line* button. Click on
+that button.
+
+.. image:: media/add-new-rules.png
+   :align: center
+   :alt: view of the rules menu, where it is possible to add new rules
+
+The available rules trigger various actions. If Odoo offers *Push* and
+*Pull* rules, others are also available. Here are some explanations of
+all of them:
+
+-  **Pull From**: this rule is triggered by a need for the product
+   in a specific stock location. The need can come from a sale order
+   which has been validated or for a manufacturing order which
+   requires a specific component. When the need appears in the
+   source location, Odoo generates a picking to fulfill this need.
+
+-  **Push To**: this rule is triggered by the arrival of some
+   products in the defined source location. In case you move
+   products to the source location, Odoo generates a picking to move
+   those products to the destination location.
+
+-  **Push & Pull**: it allows to generate pickings in the two
+   different situations explained above. It means when products are
+   required in a specific source location as well as when they are
+   pushed to a specific source location.
+
+-  **Buy**: when products are needed at the source location, a
+   request for quotation is created to fulfill the need.
+
+-  **Manufacture**: when products are needed in the source location,
+   a manufacturing order is created to fulfill the need.
+
+.. image:: media/pull-from-rule-stock-to-packing.png
+   :align: center
+   :alt: overview of a transfer requested between the stock and the packing zone
+
+With the action your rule tiggers comes the *Operation Type*. This
+operation allows defining which kind of picking is created from the
+rule. With our above screenshot, the created picking is, for example, an
+internal transfer.
+
+In the other hand, the *Supply Method*, allows defining what happens at
+the source location:
+
+-  **Take From Stock**: the products are taken from the available
+   stock of the source location;
+
+-  **Trigger Another Rule**: the system tries to find a stock rule
+   to bring the products in the source location. The available stock
+   is ignored.
+
+-  **Take From Stock, if Unavailable, Trigger Another Rule**: the
+   products are taken from the available stock of the source
+   location. If there is no stock available, the system tries to
+   find a rule to bring the products in the source location.
+
+In the *Scheduling* section, you can determine how Odoo behaves when
+one of the pickings of the chain is rescheduled. In case you decide to
+**Propagate Rescheduling**, the next move is also to be rescheduled.
+In case you prefer having the choice to manually reschedule the next
+move, you can decide to receive an alert about it under the form of a
```

Comment:
```suggestion
In case you prefer having the choice to reschedule the next move
manually, you can decide to receive an alert in the form of a
```

Reply:
"of a next activity"
--> might look better if it's in bold or italics. Otherwise, it looks a bit odd.